server/src/hardware/output/output.cpp

```diff
@@ -35,11 +35,14 @@ Output::Output(std::shared_ptr<OutputController> outputController, OutputChannel
   : interface{this, "interface", std::move(outputController), PropertyFlags::Constant | PropertyFlags::NoStore | PropertyFlags::ScriptReadOnly}
   , channel{this, "channel", channel_, PropertyFlags::Constant | PropertyFlags::NoStore | PropertyFlags::ScriptReadOnly}
   , type{this, "type", type_, PropertyFlags::Constant | PropertyFlags::NoStore | PropertyFlags::ScriptReadOnly}
+  , onValueChangedGeneric{*this, "on_value_changed_generic", EventFlags::Scriptable}
```

Comment:
`EventFlags::Scriptable` can be removed, in Lua the `onValueChanged` events can be used, in Lua you can use the same handler for different output's as Lua has weak typing.

Changing this will also fix the failing "Lua manual" build :)

Reply:
Done!